src/transport/SessionManager.h

```diff
@@ -362,6 +362,15 @@ class DLL_EXPORT SessionManager : public TransportMgrDelegate, public FabricTabl
      */
     bool MarkSessionsAsDefunct(const ScopedNodeId & node, const Optional<Transport::SecureSession::Type> & type);
 
+    /**
+     * @brief
+     *   Update all active sessions that match `node` with the provided transport peer address.
```

Comment:
```suggestion
     *   Update all CASE sessions that match `node` with the provided transport peer address.
```

Reply:
I changed this to "all active CASE" sessions right now until I better understand why we would do an address lookup on defunct sessions as well